src/dato/lib/server.clj

```diff
@@ -48,39 +52,35 @@
   {:namespace :server
    :sender    :server
    :event     event-name
+   :op        event-name
    :data      data})
 
-(defonce all-sessions
+(defonce all-sessions ^{:doc "Map of session id to session data"}
   (atom {}))
 
-(defn dato-session [session]
-  (get-in @all-sessions [(:id session)]))
-
-(defn session-server [session]
-  (let [full-session (if (:live session)
-                       session
-                       (dato-session session))
-        server (get-in full-session [:live :dato :server])]
-    (if (var? server)
-      @server
-      server)))
-
-(defn session-routes [session]
-  (let [server (session-server session)
-        routes (:routing-table server)]
-    (if (var? routes)
-      @routes
-      routes)))
-
-(defn session-db [session]
-  (let [server (session-server session)]
-    (def -session-db-session session)
-    (def -session-db-server server)
-    (ddb server)))
+(defn all-channel-ids [session-store]
+  (keys @session-store))
+
+(defn get-session [session-store session-id]
+  (get @session-store session-id))
 
 (defn serializable-sessions [all]
   (reduce into (map (fn [[k v]] {k (dissoc v :live)}) all)))
 
+(defn unwrap-var [maybe-var]
```

Comment:
I think it's important to know that it's a var we're deref-ing and not just any generic deref-able thing.